ops/travis/helpers/test_for_changes

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
+set -e
 # Usage:
-# test_for_changes ^web\/server webserver blah
+# test_for_changes ^web/server webserver blah
 # this will test if these regexp are part of the modified files since the branching,
 # or on the push commit range if running on travis
 
-REGEXPS_TO_CHECK=("$@")
-
 # detect travis
 if [ -z $TRAVIS ] || [ $TRAVIS = "false" ];
 then
     CHANGES=$(git --no-pager diff --name-only FETCH_HEAD $(git merge-base FETCH_HEAD master))
 else
-    CHANGES=$(git --no-pager diff --name-only $TRAVIS_COMMIT_RANGE) 
-fi    
+    CHANGES=$(git --no-pager diff --name-only $TRAVIS_COMMIT_RANGE)
+fi
 
 # look for changes
-for i in "${REGEXPS_TO_CHECK[@]}";
-    do
-        # echo "checking if last changes contain $i regexp..."
-        if [ -n "$(grep "$i" <<< "$CHANGES")" ];
-        then
-            echo "changes detected!"
-        fi
-    done
\ No newline at end of file
+for i in .-testing/ /helpers/ "$@"; do
```

Comment:
What does this mean?
.-testing/ /helpers/ "$@"

Reply:
it means that if there are changes to any of the relevant scripts in ops/travis things have to be rebuild